pkg/relayer/session/claim.go

```diff
@@ -113,7 +113,16 @@ func (rs *relayerSessionsManager) waitForEarliestCreateClaimsHeight(
 	// The block that'll be used as a source of entropy for which branch(es) to
 	// prove should be deterministic and use on-chain governance params.
 	claimsWindowOpenBlock := rs.waitForBlock(ctx, claimWindowOpenHeight)
-
+	// TODO_MAINNET: If a relayminer is cold-started with persisted but unclaimed ("late")
```

Comment:
No need to tend to this right now, but don't we have a type for it that either listens on or receives block events that can be reused here? Would #PUC to reference it.

Reply:
I'm not sure I'm following you here. The `BlockClient` is being used to listen for events but in the scenario described here, the block client would be started after the event for the block in question was already emitted (i.e., it would never see that block event).